src/fonts/bravura_metrics.ts

```diff
@@ -4,6 +4,9 @@ export const BravuraMetrics = {
 
   stave: {
     padding: 12,
+    endPaddingMax: 10,
```

Comment:
This is the staff padding.  endPaddingMin is the minimum padding on right.  endPaddingMax is the maximum.  unalignedNotePadding is the extra padding used to calculate min width for unaligned beats.  `padding` is (and always was) the startX of the first note from the left.